content/metadata.yaml

```diff
@@ -174,8 +174,8 @@ author_info:
     twitter: anthonygitter
     email: gitter@biostat.wisc.edu
     affiliations:
-      - Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison
-      - Morgridge Institute for Research
+      - Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, Wisconsin, United States of America
+      - Morgridge Institute for Research, Madison, Wisconsin, United States of America
```

Comment:
@agitter do you have a department at the "Morgridge Institute for Research"?

Reply:
Not exactly.  It's best to leave it as you have it now.